src/main/java/gregtech/api/metatileentity/SimpleMachineMetaTileEntity.java

```diff
@@ -90,12 +91,30 @@ public boolean onWrenchClick(EntityPlayer playerIn, EnumHand hand, EnumFacing fa
             }
             if(!getWorld().isRemote) {
                 setOutputFacing(facing);
+                if (!isAllowInputFromOutputSide()) {
+                    if (getCoverCapability(CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, facing) != null ||
+                        (getCoverCapability(CapabilityFluidHandler.FLUID_HANDLER_CAPABILITY, facing) != null)) {
+                        setAllowInputFromOutputSide(true);
+                    }
+                }
```

Comment:
I don't think this should be included. Only adding cover to output side should trigger allowing input from output side.